### ISSUE TITLE
Mining information in Dashboard missing.

### ISSUE THREAD

**Issue by astronyu** [RainbowMiner/RainbowMiner#1728]
@RainbowMiner, has the dashboard been updated? It doesn't show the mining details anymore such as hashrates etc? It just showing device status, system info and balances. 

Now, to see the information on mining I need to check on console tab? Is this normal or something is wrong on my end?

Lastly, may I ask if this is a problem? 
System.Management.Automation.RemoteException

`[18:04:41] INFO - ethash - New job: eth-sg.flexpool.io:5555, ID: f89b02ba, HEIGHT: 13229665, DIFF: 4.000G
System.Management.Automation.RemoteException`

**Comment by RainbowMiner** [comment]
No, the Dashboards has not been updated recently. All should be there. Try to click on the headlines "Running Miners on ...", "Remote Workers" etc. . Maybe the boxes are just hidden (which you can achieve by clicking on those headlines).

**Comment by RainbowMiner** [comment]
> Lastly, may I ask if this is a problem?
> System.Management.Automation.RemoteException

Which miner throws that exception? Is it Linux or Windows?

**Comment by astronyu** [comment]
> > Lastly, may I ask if this is a problem?
> > System.Management.Automation.RemoteException
> 
> Which miner throws that exception? Is it Linux or Windows?

Ok, for some reason the mining information is back.

The exception was from NBMiner mining on Flexpool, RBM is running on Windows.

**Comment by RainbowMiner** [comment]
The web UI was rewritten on 2026-07-15 (d4c520a7) and now uses Tabulator 6.3.1 instead of the old table code, so the dashboard described here no longer exists in this form. Closing as obsolete for the current master (v5.0.2.1) - please open a new issue if this still occurs. Thanks for the report!
